src/components/FolderList.jsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from "react";
+import { getFolderInfo } from "../api/api";
+import Button from "./Button";
+import styles from "../css/FolderList.module.css";
+import plusIcon from "../images/Icon_plus.svg";
+import shareIcon from "../images/Icon_share.svg";
+import editIcon from "../images/Icon_edit.svg";
+import deleteIcon from "../images/Icon_delete.svg";
+
+function FolderList() {
+  const [folderListData, setFolderListData] = useState([]);
+
+  const getFolderData = async (path) => {
+    const { data } = await getFolderInfo(path);
+    
+    if (!data) return;
+
+    setFolderListData(data);
+  }
+
+  useEffect(() => {
+    getFolderData('folders');
+  }, [])
+
+  return (
+    <>
+    <div className={styles.content}>
+      <div className={styles.container}>
+        <div>
+          <Button type="button" key={0}>전체</Button>
+          {folderListData.map((item) => {
+            return (
+                <Button type="button" key={item.id}>{item.name}</Button>
+            )
+          })}
```

Comment:
```suggestion
          {folderListData.map((item) => 
                <Button type="button" key={item.id}>{item.name}</Button>
           )}
```
1줄 리턴이실 때는 { } 없이 바로 리턴해주세요!